code/api/src/modules/product/model.js

```diff
@@ -2,7 +2,7 @@
 
 // Product
 module.exports = function(sequelize, DataTypes) {
-  return sequelize.define('products', {
+  let Product = sequelize.define('products', {
```

Comment:
Changes to add belongs to many relationship to products connecting it to joins table